Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.11-alpine3.7 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
+RUN apk add vips-dev --verbose --update-cache --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ --repository https://dl-3.alpinelinux.org/alpine/edge/main
+
+ADD . /src
+WORKDIR /src
+RUN make dep
+RUN make test
+RUN make build
```

Comment:
`make` runs `make test` and `make build`.

Reply:
Sorry it's a residue from when I start writing this Dockerfile.
`make` test was broken so I have add explicit call to `make build` and recently readded `make test`